### ISSUE TITLE
Feature Request: gPodder and a little bit of context

### ISSUE THREAD

**Issue by JOduMonT** [airsonic/airsonic#1811]
Airsonic rock;

> I cross open this Feature Request at Airsonic, [Audinaut](https://github.com/nvllsvm/Audinaut/issues/97), [Clementine](https://github.com/clementine-player/Clementine/issues/6811), and [Ultrasonic](https://github.com/ultrasonic/ultrasonic/issues/330) with hope of collaboration.

I start using it for my music, then I realize I could centralise my podcast too.
While on my Android, I'm juggling, between Subsonic, Audinaut, Ultrasonic and AntennaPod
and on my Laptop using Clementine

I still see a place for improvement
- **Audinaut** is blazing fast but doesn't support podcast and video. It also misses function like Recently added , Top rated and so on I could find in Subsonic. I mainly use it to listen my music.
- **Ultrasonic** support podcast but, it is slowish and unstable/don't work while I try to use is for video. It also misses function like Recently added, Top rated and so on we could find in Subsonic. I was using it for my podcast, but I transferred to AntennaPod.
- **Subsonic**: I only use it for the missing function in Audinaut and Ultrasonic.
- **AntennaPod**: does what it has to do, syncing your podcast with gPodder, but nothing else.
- **Clementine**: technically able to use the Subsonic API to connect with airsonic but I don't see my podcast.

Sadly, I'm not technical enough to understand and see what/if it is possible but

- if airsonic as a gPodder plugin for the podcast it would be awesome
- if Audinaut was able to sync/play podcast and/or a way to sync with gPodder it would be marvelous
- if Ultrasonic was an option to sync with gPodder it would love it more
- if Clementine was able to sync/play podcast I have on my airsonic, it would be great.

So basically right now, I'm using the podcast option on my airsonic simply to republish and share them, but I have to manually add my new podcast I discover via AntennaPod.

**Comment by kromsam** [comment]
+1 for gPodder support in AirSonic
